ydb/public/sdk/cpp/include/ydb-cpp-sdk/client/federated_topic/federated_topic.h

```diff
@@ -212,34 +217,43 @@ struct TReadSessionEvent {
 //! Set of offsets to commit.
 //! Class that could store offsets in order to commit them later.
 //! This class is not thread safe.
-class TDeferredCommit {
+class TDeferredCommit: NTopic::TDeferredCommit {
```

Comment:
This is extremely quick band-aid, I have not even tried to ensure it works properly